### PR TITLE
[TRANSLATION] Update Chinese shortcut translation 

### PR DESCRIPTION
**Pull request by njlyf2011**

### CODE REVIEW

**Comment by njlyf2011** [comment]
Cc @katahiromz 
I did not find a suitable Chinese word to translate the name of Font Substitutes Editor. It is better to use the English name than to translate it into Chinese, so I have not translated it.

**Comment by katahiromz** [comment]
Google translated: 字體替代編輯器

**Comment by njlyf2011** [comment]
I have considered this translation, but I am worried that others will misunderstand the word "字体替代".

**Comment by katahiromz** [comment]
How about 字体名替代？

**Comment by njlyf2011** [comment]
Added and updated.

File: modules/rosapps/applications/sysutils/fontsub/lang/zh-CN.rc [hunk 1]
```diff
@@ -73,7 +73,7 @@ BEGIN
     IDS_QUERYDELETE,    "您确定要删除此项目吗？"
     IDS_CANTOPENKEY,    "无法打开注册表键值。"
     IDS_REBOOTNOW,      "注册表已经更新。现在需要重启系统吗？"
-    IDS_ABOUT,          "FontSub (Font Substitute Editor) Version 0.5\r\nby Katayama Hirofumi MZ and The ReactOS Team\r\n\r\n本软件基于 CC0 1.0 许可证发布。"
+    IDS_ABOUT,          "FontSub (字体名称替代) 版本 0.5\r\nby Katayama Hirofumi MZ 和 The ReactOS Team\r\n\r\n本软件基于 CC0 1.0 许可证发布。"
```
Comment (katahiromz) [inline modules/rosapps/applications/sysutils/fontsub/lang/zh-CN.rc]: Can you translate "Katayama Hirofumi MZ"?

Comment (learn-more) [reply modules/rosapps/applications/sysutils/fontsub/lang/zh-CN.rc]: @katahiromz You can add a suggestion here, using this button:
test
This way the author can simply integrate your suggestion in the PR.

**Comment by njlyf2011** [comment]
Sorry, I forgot to translate the Simplified Chinese entries.

File: modules/rosapps/applications/sysutils/fontsub/lang/zh-CN.rc (hunk 1 above)
Comment (katahiromz) [reply modules/rosapps/applications/sysutils/fontsub/lang/zh-CN.rc]:
```suggestion
    IDS_ABOUT,          "FontSub (字体名称替代) 版本 0.5\r\nby 片山博文MZ 和 The ReactOS Team\r\n\r\n本软件基于 CC0 1.0 许可证发布。"
```

**Comment by njlyf2011** [comment]
Corrected.

**Comment by katahiromz** [comment]
Waiting for checks...

**Pull request merged**